tests/integration/nm/linux_bridge_test.py

```diff
@@ -179,8 +178,7 @@ def _attach_port_to_bridge(port_state):
     port_con_profile.create(iface_port_settings)
 
     curr_port_con_profile.update(port_con_profile)
-    curr_port_con_profile.commit(nmdev=port_nmdev)
-    nm.device.modify(port_nmdev, curr_port_con_profile.profile)
+    nm.device.modify(port_nmdev, port_con_profile.profile)
```

Comment:
Same here. Looks to me like it will activate a new profile instead of reapplying the old one.

Reply:
the `Update2` is __replacing__ old profile with new. Activating/Reapplying the old one will cause system revert back to old state.

Reply:
Reapply is for device, not for profile.